SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -0,0 +1,116 @@
+%global debug_package %{nil}
+Summary:        NVIDIA container runtime hook
+Name:           nvidia-container-toolkit
+Version:        1.4.2
+Release:        2%{?dist}
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/nvidia-container-runtime
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        nvidia-container-toolkit-1.4.2.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar -cf %%{name}-%%{version}-vendor.tar.gz vendor
+Source1:        %{name}-%{version}-vendor.tar.gz
+Patch0:         nvidia-container-toolkit-1.4.2.patch
+BuildRequires:  golang
+Provides:       nvidia-container-runtime-hook
+
+%description
+Provides a OCI hook to enable GPU support in containers.
+
+%prep
+%setup -q
+%patch0 -p1
+tar -xvf %{SOURCE1}
+
+%build
+go build -ldflags "-s -w " -o "nvidia-container-toolkit" ./pkg
+
+%install
+mkdir -p %{buildroot}%{_bindir}
+install -m 755 -t %{buildroot}%{_bindir} nvidia-container-toolkit
+
+cp config/config.toml.centos config.toml
+mkdir -p %{buildroot}%{_sysconfdir}/nvidia-container-runtime
+install -m 644 -t %{buildroot}%{_sysconfdir}/nvidia-container-runtime config.toml
+
+mkdir -p %{buildroot}%{_libexecdir}/oci/hooks.d
+install -m 755 -t %{buildroot}%{_libexecdir}/oci/hooks.d oci-nvidia-hook
+
+mkdir -p %{buildroot}%{_datadir}/containers/oci/hooks.d
+install -m 644 -t %{buildroot}%{_datadir}/containers/oci/hooks.d oci-nvidia-hook.json
+
+%posttrans
+ln -sf %{_bindir}/nvidia-container-toolkit %{_bindir}/nvidia-container-runtime-hook
```

Comment:
Do we need both names? Why?

Reply:
What names are you referring to here is it nvidia-container-toolkit and nvidia-container-runtime-hook. If so i believe this is for dependencies that other Nvidia binaries have and  they use the nvidia-container-runtime-hook path.